code/restful-api/restful-api/Models/Realiza.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RestfulApi.Models
+{
+    public class Realiza
```

Comment:
Why is this class needed? Since it is an 1-1 relationship, as one `Compra` is paid with only one `CartaoDeCredito`.

Reply:
Excluido a class Realiza

Reply:
🆗